src/glang/lib/std/io.c3

```diff
@@ -110,7 +121,7 @@ function puts : (str : StringView) -> int = {
     // Otherwise, it shall return EOF,
     // TODO: shall set an error indicator for the stream,
     // TODO: and errno shall be set to indicate the error.
-    return EOF();
```

Comment:
Why?

Reply:
Hmm, yeah. Maybe this should be a separate commit. It felt like posix was leaking into one of our non-posix headers.

Reply:
Yeah I wrote `puts` based on POSIX. Still not sure whether we should provide a POSIX-like API or not